tests/Unit/ExampleEventTest.php

```diff
@@ -75,6 +75,30 @@ public function testSerializeEncodeDecode()
         );
     }
 
+    /**
+     * Test that serialization and deserialization return the same result (but with @ prefixes added)
+     * after the process.
+     *
+     * @return void
+     */
+    public function testNonPrefixedSerializeEncodeDecode()
+    {
+        $original = "{\"@context\":[\"https:\\/\\/openactive.io\\/\",\"https:\\/\\/openactive.io\\/ns-beta\"],\"type\":\"Concept\",\"id\":\"https://openactive.io/facility-types#37bbed12-270b-42b1-9af2-70f0273990dd\",\"prefLabel\":\"Grass\",\"inScheme\":\"https://openactive.io/facility-types\"}";
+        $expected = "{\"@context\":[\"https:\\/\\/openactive.io\\/\",\"https:\\/\\/openactive.io\\/ns-beta\"],\"@type\":\"Concept\",\"@id\":\"https://openactive.io/facility-types#37bbed12-270b-42b1-9af2-70f0273990dd\",\"prefLabel\":\"Grass\",\"inScheme\":\"https://openactive.io/facility-types\"}";
+
+        $decode = Concept::deserialize($original);
+        $encode = Concept::serialize($decode);
+
+        // $this->assertSame(
+        //     "https://openactive.io/facility-types#37bbed12-270b-42b1-9af2-70f0273990dd",
+        //     $decode->getId()
+        // );
```

Comment:
Is it worth removing this? I believe it was commented out in the original .NET tests (https://github.com/openactive/OpenActive.NET/blob/master/OpenActive.NET.Test/EventTest.cs#L251)